project/pom.xml

```diff
@@ -361,6 +362,18 @@
         <artifactId>java-xmlbuilder</artifactId>
         <version>${java-xmlbuilder.version}</version>
       </dependency>
+      <dependency>
```

Comment:
Wondering if this adds anything here? `sshj` is only declared as a dependency in one library. If anything, the version that keeps reappearing is the `jsch.agentproxy` version, not the sshj version.